cmd/authgear/config/marshal.go

```diff
@@ -3,21 +3,24 @@ package config
 import (
 	"fmt"
 	"os"
+	"path/filepath"
 
 	"sigs.k8s.io/yaml"
 )
 
-func MarshalConfigYAML(cfg interface{}, outputPath string) error {
+func MarshalConfigYAML(cfg interface{}, outputFolderPath string, fileName string) error {
 	yaml, err := yaml.Marshal(cfg)
 	if err != nil {
 		return err
 	}
 
-	if outputPath == "-" {
+	if outputFolderPath == "-" {
 		_, err = os.Stdout.Write(yaml)
 		return err
 	}
 
+	outputPath := filepath.Join(outputFolderPath, fileName)
```

Comment:
The command should be smart enough to create the directory if it does not exist. Use os.MkdirAll to mimic `mkdir -p` in case the argument has more than 1 level of directories.